<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCuentasIBM</class>
 <widget class="QDialog" name="frmCuentasIBM">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>675</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Estudio de la Cuenta seleccionada</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images/xulpymoney.qrc">
    <normaloff>:/xulpymoney/report2.png</normaloff>:/xulpymoney/report2.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblTitulo">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 128, 0);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tab">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tabDatos">
        <attribute name="title">
         <string>Datos de la Cuenta</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Selecciona la entidad bancaria a la que pertenece</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbEB"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Nombre de la Cuenta</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtCuenta">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Seleccione la divisa de la cuenta</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbCurrency"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Número de cuenta</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtNumero">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QCheckBox" name="chkActiva">
               <property name="text">
                <string>¿Está activa?</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="cmdDatos">
             <property name="text">
              <string>Actualizar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabMovimientos">
        <attribute name="title">
         <string>Movimientos de la cuenta</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Selecciona el mes y el año de los datos a visualizar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbMonth">
               <item>
                <property name="text">
                 <string>Enero</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Febrero</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Marzo</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Abril</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mayo</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Junio</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Julio</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Agosto</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Septiembre</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Octubre</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Noviembre</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Diciembre</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbYear"/>
             </item>
             <item>
              <widget class="QToolButton" name="cmdMovimientos">
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="../images/xulpymoney.qrc">
                 <normaloff>:/xulpymoney/search.png</normaloff>:/xulpymoney/search.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="myQTableWidget" name="tblOperaciones">
             <property name="contextMenuPolicy">
              <enum>Qt::CustomContextMenu</enum>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Fecha</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Concepto</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Importe</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Saldo</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Comentario</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabTarjetas">
        <attribute name="title">
         <string>Tarjetas asociadas</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>Listado de tarjetas asociadas a la cuenta</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QCheckBox" name="chkTarjetas">
                  <property name="text">
                   <string>¿Mostrar tarjetas inactivas?</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="myQTableWidget" name="tblTarjetas">
                  <property name="contextMenuPolicy">
                   <enum>Qt::CustomContextMenu</enum>
                  </property>
                  <property name="alternatingRowColors">
                   <bool>true</bool>
                  </property>
                  <property name="selectionMode">
                   <enum>QAbstractItemView::SingleSelection</enum>
                  </property>
                  <property name="selectionBehavior">
                   <enum>QAbstractItemView::SelectRows</enum>
                  </property>
                  <attribute name="verticalHeaderVisible">
                   <bool>false</bool>
                  </attribute>
                  <column>
                   <property name="text">
                    <string>Tarjeta</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Número</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Activa</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Pago diferido</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Saldo máximo</string>
                   </property>
                  </column>
                  <column>
                   <property name="text">
                    <string>Saldo</string>
                   </property>
                  </column>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="layDiferido">
             <item>
              <widget class="QTabWidget" name="tabOpertarjetasDiferidas">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="tab_2">
                <attribute name="title">
                 <string>Operaciones diferidas de la tarjeta seleccionada</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QGroupBox" name="grpOperTarjetas">
                   <property name="title">
                    <string>Operaciones diferidas de la tarjeta seleccionada</string>
                   </property>
                   <property name="checkable">
                    <bool>false</bool>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_7">
                    <item>
                     <widget class="myQTableWidget" name="tblOperTarjetas">
                      <property name="contextMenuPolicy">
                       <enum>Qt::CustomContextMenu</enum>
                      </property>
                      <property name="editTriggers">
                       <set>QAbstractItemView::NoEditTriggers</set>
                      </property>
                      <property name="alternatingRowColors">
                       <bool>true</bool>
                      </property>
                      <property name="selectionMode">
                       <enum>QAbstractItemView::MultiSelection</enum>
                      </property>
                      <property name="selectionBehavior">
                       <enum>QAbstractItemView::SelectRows</enum>
                      </property>
                      <attribute name="verticalHeaderVisible">
                       <bool>false</bool>
                      </attribute>
                      <column>
                       <property name="text">
                        <string>Fecha</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Concepto</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Importe</string>
                       </property>
                      </column>
                      <column>
                       <property name="text">
                        <string>Comentario</string>
                       </property>
                      </column>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="grpPago">
                   <property name="title">
                    <string>Pago diferido de operaciones seleccionadas</string>
                   </property>
                   <layout class="QHBoxLayout" name="horizontalLayout_11">
                    <item>
                     <layout class="QVBoxLayout" name="verticalLayout_8">
                      <item>
                       <widget class="QLabel" name="lblPago">
                        <property name="font">
                         <font>
                          <pointsize>12</pointsize>
                          <weight>75</weight>
                          <bold>true</bold>
                         </font>
                        </property>
                        <property name="text">
                         <string>XXXX €</string>
                        </property>
                        <property name="alignment">
                         <set>Qt::AlignCenter</set>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <layout class="QHBoxLayout" name="horizontalLayout_10">
                        <item>
                         <widget class="QLabel" name="label_5">
                          <property name="text">
                           <string>Selecciona la fecha de pago</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QDateEdit" name="calPago">
                          <property name="displayFormat">
                           <string>yyyy/MM/dd</string>
                          </property>
                          <property name="calendarPopup">
                           <bool>true</bool>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                      <item>
                       <widget class="QPushButton" name="cmdPago">
                        <property name="text">
                         <string>Pagar</string>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="tab_3">
                <attribute name="title">
                 <string>Histórico de pagos diferidos de la tarjeta seleccionada</string>
                </attribute>
                <layout class="QHBoxLayout" name="horizontalLayout_14">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_9">
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_13">
                     <item>
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>Selecciona un pago</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QComboBox" name="cmbFechasPago"/>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>Para devolver el pago</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="cmdDevolverPago">
                       <property name="text">
                        <string>Devolver el pago</string>
                       </property>
                       <property name="icon">
                        <iconset>
                         <normaloff>:/xulpmoney/tools-wizard.png</normaloff>:/xulpmoney/tools-wizard.png</iconset>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <widget class="myQTableWidget" name="tblOpertarjetasHistoricas">
                     <property name="alternatingRowColors">
                      <bool>true</bool>
                     </property>
                     <property name="selectionMode">
                      <enum>QAbstractItemView::SingleSelection</enum>
                     </property>
                     <property name="selectionBehavior">
                      <enum>QAbstractItemView::SelectRows</enum>
                     </property>
                     <attribute name="verticalHeaderVisible">
                      <bool>false</bool>
                     </attribute>
                     <column>
                      <property name="text">
                       <string>Id</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>Fecha</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>Concepto</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>Importe</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>Saldo</string>
                      </property>
                     </column>
                     <column>
                      <property name="text">
                       <string>Comentario</string>
                      </property>
                     </column>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionMovimientoNuevo">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo movimiento</string>
   </property>
  </action>
  <action name="actionMovimientoBorrar">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/list-remove.png</normaloff>:/xulpymoney/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Borrar el movimiento</string>
   </property>
  </action>
  <action name="actionTarjetaNueva">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>Nueva Tarjeta</string>
   </property>
  </action>
  <action name="actionTarjetaBorrar">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/list-remove.png</normaloff>:/xulpymoney/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Borrar Tarjeta</string>
   </property>
  </action>
  <action name="actionOperTarjetaNueva">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/add.png</normaloff>:/xulpymoney/add.png</iconset>
   </property>
   <property name="text">
    <string>Nueva Operación de Tarjeta</string>
   </property>
  </action>
  <action name="actionOperTarjetaBorrar">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/list-remove.png</normaloff>:/xulpymoney/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Borrar Operación de Tarjeta</string>
   </property>
  </action>
  <action name="actionTarjetaModificar">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/editar.png</normaloff>:/xulpymoney/editar.png</iconset>
   </property>
   <property name="text">
    <string>Modificar Tarjeta</string>
   </property>
   <property name="toolTip">
    <string>Modificar Tarjeta</string>
   </property>
  </action>
  <action name="actionTarjetaActivar">
   <property name="text">
    <string>ActivarTarjeta</string>
   </property>
  </action>
  <action name="actionMovimientoModificar">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/editar.png</normaloff>:/xulpymoney/editar.png</iconset>
   </property>
   <property name="text">
    <string>Modificar movimiento</string>
   </property>
   <property name="toolTip">
    <string>Modificar Movimiento</string>
   </property>
  </action>
  <action name="actionOperTarjetaModificar">
   <property name="icon">
    <iconset resource="../images/xulpymoney.qrc">
     <normaloff>:/xulpymoney/editar.png</normaloff>:/xulpymoney/editar.png</iconset>
   </property>
   <property name="text">
    <string>Modificar Operación de Tarjeta</string>
   </property>
   <property name="toolTip">
    <string>Modificar Operación de Tarjeta</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">myqtablewidget</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
